openedx/core/djangoapps/user_api/accounts/views.py

```diff
@@ -324,6 +324,8 @@ def list(self, request):
                 user = User.objects.get(id=lms_user_id)
             except (UserNotFound, User.DoesNotExist):
                 return Response(status=status.HTTP_404_NOT_FOUND)
+            except ValueError:
+                return Response(status=status.HTTP_400_BAD_REQUEST)
```

Comment:
what other options do we have for return status code?

Reply:
<img width="552" alt="Screenshot 2021-12-15 at 5 17 51 PM" src="https://user-images.githubusercontent.com/52413434/146185295-175d3bc1-95a7-453c-9d4f-2a31c29d9a38.png">